src/Zip.Shared/WinZipAes.cs

```diff
@@ -356,7 +356,7 @@ internal class WinZipAesCipherStream : Stream
 
         internal HMACSHA1 _mac;
 
-        internal AesManaged _aesCipher;
+        internal Aes _aesCipher;
```

Comment:
I think now, as we can no longer rely on Aes implementation to be fully in managed code, we will have to dispose _aesCipher in order not to cause memoryleaks.

Reply:
sorry, didn't think of that, will have a look.

(actually, does it need to keep _aesCipher as a member, or can just _xform be kept? still need to dispose that though)

Reply:
Not sure if it matters, but I think _xform should be disposed before _aesCipher, because _xform  is originally created from _aesCipher (parent-child).

Reply:
not sure, but i swapped them over in any case